docs/getting-started/typescript-composition-api.md

```diff
@@ -10,8 +10,6 @@ position: 20
 
```

Comment:
See the comments in `docs/getting-started/javascript-options-api.md` as they are valid for this file as well.